lighthouse-viewer/gulpfile.js

```diff
@@ -40,118 +57,113 @@ gulp.task('lint', () => {
 
 gulp.task('images', () => {
   return gulp.src('app/images/**/*')
-  .pipe(gulp.dest(`${config.dist}/images`));
+  .pipe(gulp.dest(`dist/images`));
 });
 
 gulp.task('concat-css', () => {
-  return gulp.src([
-    '../lighthouse-core/report/styles/report.css',
-    '../lighthouse-core/report/partials/*.css',
-    'app/styles/viewer.css',
-  ])
-  .pipe($.concat('viewer.css'))
-  .pipe(gulp.dest(`${config.dist}/styles`));
+  const reportCss = streamFromString(ReportGenerator.reportCss, 'report-styles.css');
+  const viewerCss = gulp.src('app/styles/viewer.css');
+
+  return streamqueue({objectMode: true}, reportCss, viewerCss)
```

Comment:
is most of this to get around `require()` usage? Is there a reason you got ride of require?